airplanedb.py

```diff
@@ -26,17 +26,156 @@ def reset_db(self):
         cursor = self.airdb.cursor()
 
         drop = 'DROP TABLE IF EXISTS {}'
-        cursor.execute(drop.format('USERS'))
+        cursor.execute(drop.format('SCHEDULE'))
+        cursor.execute(drop.format('WORKSON'))
+        cursor.execute(drop.format('FREQUENTFLIER'))
+        cursor.execute(drop.format('ITINERARY'))
+        cursor.execute(drop.format('EMPLOYEE'))
+        cursor.execute(drop.format('FLIGHT'))
+        cursor.execute(drop.format('AIRCRAFT'))
+        cursor.execute(drop.format('GATE'))
+        cursor.execute(drop.format('AIRPORT'))
+        cursor.execute(drop.format('BAGGAGE'))
+        cursor.execute(drop.format('CUSTOMER'))
 
-        raw_users_query = """CREATE TABLE USERS (
-                             U_ID INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
-                             U_NAME VARCHAR(32),
-                             U_AGE INT,
-                             U_HOME_ADDR VARCHAR(128),
-                             U_PHONE_NUMBER VARCHAR(32) )"""
-        cursor.execute(raw_users_query)
-        print(('Created new {0} table in {1}').format('USERS',self.db))
+        create_customer_table = """CREATE TABLE CUSTOMER (
```

Comment:
can you fix the whitespace but converting all your tabs to spaces (1 tab = 4 space alignment) thanks

Reply:
Did this fix the spacing issue?

Reply:
looks like it, thanks! & good job on fixing the cursor bug too